node/src/container_chain_spawner.rs

```diff
@@ -235,6 +246,8 @@ async fn try_spawn(
     );
 
     if !start_collation {
+        collation_params = None;
```

Comment:
I forgot why we used both `validator && start_collation` instead of just one boolean, here is a summary:

The reason `start_collation` exists as a separate arg to `validator` is that in the old days instead of restarting the container chain node we used `collate_on` to sent a message to enable the collator. So we would have to pass `collation_params` set to `Some` and `start_collation` set to false, and then use the `collate_on` closure to start collation later.

Now we don't have that `collate_on`, so just setting to `None` here is ok. Although I'm considering bringing `collate_on` back, for parathreads, but that's an issue for the future.

Reply:
mayeb we should add a comment for this then

Reply:
I can do that in the next refactor